bigquery/google/cloud/bigquery/table.py

```diff
@@ -531,23 +550,7 @@ def view_query(self):
         """Delete SQL query defining the table as a view."""
         self._properties.pop('view', None)
 
-    @property
-    def view_use_legacy_sql(self):
-        """Specifies whether to execute the view with Legacy or Standard SQL.
-
-        The default is False for views (use Standard SQL).
-        If this table is not a view, None is returned.
-
-        :rtype: bool or ``NoneType``
-        :returns: The boolean for view.useLegacySql, or None if not a view.
-        """
-        view = self._properties.get('view')
-        if view is not None:
-            # useLegacySql is never missing from the view dict if this table
-            # was created client-side, because the view_query setter populates
-            # it. So a missing or None can only come from the server, whose
-            # default is True.
-            return view.get('useLegacySql', True)
+    view_use_legacy_sql = property(_view_use_legacy_sql_getter)
```

Comment:
You can keep the docstring, just dedent it 4 spaces?

Reply:
The docs render fine when I test it out. It grabs them off the `_view_use_legacy_sql_getter`.

Reply:
![image](https://user-images.githubusercontent.com/247555/33044261-1b5559c2-cdfd-11e7-8607-c9ff840c4460.png)

Reply:
Ah OK. I wonder why that is? (I hope it's not a strange Sphinx cache "feature" making it seem like that.)